packages/core/src/utils.ts

```diff
@@ -39,6 +38,21 @@ export function isBrowser() {
   );
 }
 
+function isBuffer(obj: any): boolean {
```

Comment:
does this essentially check whether `obj` is a byte array? My JS is a bit rusty.

Reply:
No, just a nodejs Buffer.
The reason we can't use Buffer.isBuffer is that this code must be compatible with browsers where `Buffer` isn't a concept